logic/calendar.py

```diff
@@ -0,0 +1,35 @@
+import urllib.request, json
+from os import getenv
+
+class CalendarImpl:
+    def command_parse(text: str):
+        command = text.split(" ")        
+
+        title = command[0]
+        dateindex = int(command.index("-d") + 1)
+        date = command[dateindex]
+        locationindex = int(command.index("-l") + 1)
+        location = command[locationindex]
+
+        obj = {"value1" : date, "value2" : title, "value3" : location} 
+        json_data = json.dumps(obj).encode("utf-8")
+
+        return json_data
+
+    @classmethod
+    def add_event(cls, text: str):
+        message = "Success!"        
+
+        # set up POST data
+        url = getenv("CALENDAR_WEBHOOK")
+        method = "POST"
+        headers = {"Content-Type": "application/json"}
+
+        json_data= command_parse(text)
+
+        # http request
+        request = urllib.request.Request(url, data=json_data, method=method, headers=headers)
+        urllib.request.urlopen(request)
+
+        return message
```

Comment:
```suggestion
    def add_event(cls, title: str, date: str, location: str):   
        # set up POST data
        url = getenv("CALENDAR_WEBHOOK_URL")
        method = "POST"
        headers = {"Content-Type": "application/json"}
        json_data = command_parse(title, date, location)

        # http request
        request = urllib.request.Request(url, data=json_data, method=method, headers=headers)
        urllib.request.urlopen(request)
```
他の変更に合わせるとこうなるね

getenv じゃなく、environ からキーを元に取得したい所
変更が面倒なので書いてないけど